src/Input/Input.js

```diff
@@ -130,7 +130,7 @@ export default class Input extends Component {
     /**
      * The input value, required for a controlled component.
      */
-    value: PropTypes.string,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
> but does it make sense to allow arrays and objects and functions for an input?

Good point, I'm pretty convinced people we ask for it. I could ask the same question, does it makes sense to provide a number and not a string? People will try to provide boolean too.